src/lib/utils/computeRRule/computeRRule.js

```diff
@@ -0,0 +1,16 @@
+import RRule from 'rrule';
+import { assign } from 'lodash';
+
+import computeRepeat from './computeRepeat';
+import computeEnd from './computeEnd';
+
+const computeRRule = ({ repeat, end }) => {
+  const repeatObject = computeRepeat(repeat);
+  const endObject = computeEnd(end);
+
+  const rruleObject = assign({}, repeatObject, endObject);
```

Comment:
You seem to build new objects with `assing({},...)` in many palces in system... consider using deconstriction like `const rruleObject = { ...repeatObject, ...endObject };`

Reply:
Sure, I don't know why I didn't use destructuring. Probably I wasn't sure how to write this:
```js
assign(
    {},
    { freq: RRule.MONTHLY },
    { interval },
    mode === 'on' ? computeMonthlyOn(on) : computeMonthlyOnThe(onThe),
  );
```
with spread syntax.

So for now I removed all lodash'es assign(), thanks for pointing that!